docs/dataset.md

```diff
@@ -0,0 +1,22 @@
+# Dataset configuration format
+
+This document describes the format of the configuration structure used to setup the Promscale dataset options. Setup is done by using the `-startup.dataset.config` flag which should contain this configuration structure in YAML format
+
+Example usage:
+```
+promscale -startup.dataset.config=$'metrics:\n  default_chunk_interval: 6h'
+```
+
+Expected format is YAML and currently only option that can be set is default chunk interval.
+
+Example configuration in config.yaml:
+
+``
```

Comment:
Nit: Seems one ` is missing. Also worth mentioning that this is passed in string format just so end users don't get confused and try to use object syntax.